src/static/js/main.js

```diff
@@ -46,17 +46,58 @@ $(function () {
       contentType: 'application/json'
     }).done(function (response) {
       $(form).find('button[type=submit]').toggleClass('loading')
-      // This is used allow each template to implement its own response handler
-      // if (FORM_HANDLERS[$(form).attr('name')]) {
-      //   FORM_HANDLERS[$(form).attr('name')](form, response)
-      // }
+     
 
-      console.log('asdasdasd', THE_ORG_BOOK_APP_URL)
+      console.log('asdasdasd')
 
-      window.location.replace(
-        THE_ORG_BOOK_APP_URL + '/en/recipe/start_a_restaurant?record=' +
-        response.result.id
-      )
+
+      if (response.result == null) {
+        
+                // $(".message").append(" <b>Your request is being processed by one of our representative</b>.");
+
+                alert('Your request is being processed by one of our representative')
```

Comment:
Change to - "Your request has been flagged as requiring additional review by the BC Registries Service Team.  You will be notified when the review process is complete and informed of any further steps needed to complete the process."